cx_fwup

```diff
@@ -0,0 +1,12 @@
+#!/bin/env python3
+
+import os, sys
+
+def main():
+    os.system("chmod +x mlxup")
+    r = os.system("/mlxup -y")
```

Comment:
Does this update the firmware on all mlx cards to latest? What if there are multiple on the server?

Reply:
This repo is targeting DPUs. I'm wondering if this is the right home for mlxup.

Reply:
Only issue is the workers will likely have coreos on them, so I imagine we would need a separate container image to run this in.

This would allow us to reuse more code at the cost of broadening the scope of this repo. I'm fine either way

Reply:
Good point. In that case, I'm fine to add it to this repo.